src/ontology/mondo-ingest.Makefile

```diff
@@ -373,19 +373,21 @@ USE_MONDO_RELEASE=false
 
 # Builds tmp/mondo/ and rebuilds mondo.owl and mondo.sssom.tsv, and stores hash of latest commit of mondo repo main branch in tmp/mondo_repo_built
 tmp/mondo_repo_built:
+	git config --global --add safe.directory /work/src/ontology/tmp/mondo
```

Comment:
We should confirm that this is still needed once https://github.com/monarch-initiative/mondo-ingest/pull/594 is done.

Reply:
It is still needed. I believe it is because the nature of how `mondo-edit.owl`, a file which we introduce w/ this PR,  gets generated is fundamentally different in some way than `mondo.owl` and `mondo.sssom.tsv` and is what is causing this error.

So I've added these lines back. I think that is more appropriate to do in this PR rather than an independent PR because I think there is enough evidence to suggest that `mondo-edit.owl` is probably what is introducing this issue.

Reply:
We need to discuss if this is the appropriate solution for this issue. 
- Is `mondo-edit.obo` only used to extract EXCLUDED synonyms? 
- Is there already a report of the excluded synonyms generated in the mondo repo that could be used or if not, could a report be created in the mondo repo so that this git configuration change is not needed here?

Reply:
I'm not aware of any other location of these excluded synonyms other than inside `mondo-edit.obo`. I imagine one can be created in `mondo` repo and I think that'd might be better, but I'm unfamiliar w/ the `mondo` repo pipeline / best way to do that. Could ping Nico, or maybe bring to tech call?

Reply:
I'm not aware of any other location of these excluded synonyms other than inside `mondo-edit.obo`. I imagine one can be created in `mondo` repo and I think that'd might be better, but I'm unfamiliar w/ the `mondo` repo pipeline / best way to do that. Could ping Nico, or maybe bring to tech call?

Reply:
Completed in: 44c978fe2c5e4fc362d12d451b1ed8e3386146f3